components/common/product/product.js

```diff
@@ -0,0 +1,574 @@
+import React, { useEffect, useState } from "react";
+import Image from "next/image";
+import img from "./../../../public/images/product-01.jpg";
+import iconHeart1 from "./../../../public/images/icons/icon-heart-01.png";
+import iconHeart2 from "./../../../public/images/icons/icon-heart-02.png";
+import Link from "next/link";
+
+const Product = () => {
+  const [indexh, setindexh] = useState("");
+  const [hv, sethv] = useState(false);
+  const hoverImg = (index) => {
+    sethv(true);
+    setindexh(index);
+  };
+  const unhoverImg = (index) => {
+    sethv(false);
+    setindexh(index);
+  };
+
+  const productItems = [
+    {
+      img: img,
+      name: "Esprit Ruffle Shirt",
+      price: "$16,64",
+      icon1: iconHeart1,
```

Comment:
should use firstIcon, secondIcon instead of icon1 , icon 2. Replace all var name